cypress/integration/collections_spec.js

```diff
@@ -43,14 +48,9 @@ describe('Dashboard Collections Page', () => {
       cy.get('@addCollection').click();
 
       // fill the form and submit
-      const duplicateHandling = 'error';
-      const collection = {
-        name: 'TESTCOLLECTION',
-        version: '006',
-        dataType: 'TESTCOLLECTION',
-        duplicateHandling
-      };
-      cy.editJsonTextarea({ data: collection });
+      cy.fixture('TESTCOLLECTION___006.json').then((collection) => {
```

Comment:
If the only value we're updating is the duplicateHandling, what is the value of adding a fixture with a full collection object? I'm a little hesitant to use Cypress fixtures because it implies that we're using Cypress request stubbing, which we are not doing.

Reply:
It is for `add collection`.  Since now adding a full collection object is possible (thank you!), we would want to add a full collection object complying with the collection schema.
https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js#L143